lib/utils.js

```diff
@@ -928,6 +929,14 @@ function makeClientMetadata(options) {
 
 const noop = () => {};
 
+function emitDeprecatedOptionWarning(options, name, list) {
```

Comment:
do we need `name` here? [process.emitWarning](https://nodejs.org/api/process.html#process_process_emitwarning_warning_options) will provide a `stack` property if a user listens to `process.on('warning', ...)`, so they should have the details they need to figure out where the option is passed in. 

Also, can you please document this method and its inputs?

Reply:
removed name, updated emitDeprecatedOptionWarning